x-pack/plugins/security_solution/public/management/components/artifact_list_page/components/no_data_empty_state.test.ts

```diff
@@ -60,51 +61,97 @@ describe('When showing the Empty State in ArtifactListPage', () => {
     expect(renderResult.queryByTestId('header-page-title')).toBe(null);
   });
 
-  it('should open create flyout when primary button is clicked', async () => {
-    render();
-    const addButton = await renderResult.findByTestId('testPage-emptyState-addButton');
+  describe('and user is allowed to Create entries', () => {
+    it('should show title, about info and add button', async () => {
+      const { getByTestId, queryByTestId } = render();
 
-    act(() => {
-      userEvent.click(addButton);
-    });
+      await waitFor(async () => {
+        expect(getByTestId('testPage-emptyState'));
+      });
 
-    expect(renderResult.getByTestId('testPage-flyout')).toBeTruthy();
-    expect(history.location.search).toMatch(/show=create/);
-  });
+      expect(getByTestId('testPage-emptyState-title').textContent).toEqual(
+        artifactListPageLabels.emptyStateTitle
+      );
+      expect(getByTestId('testPage-emptyState-aboutInfo').textContent).toEqual(
+        artifactListPageLabels.emptyStateInfo
+      );
+      expect(getByTestId('testPage-emptyState-addButton').textContent).toEqual(
+        artifactListPageLabels.emptyStatePrimaryButtonLabel
+      );
+
+      expect(queryByTestId('testPage-emptyState-title-no-entries')).toBeNull();
+    });
 
-  describe('and the first item is created', () => {
-    it('should show the list after creating first item and remove empty state', async () => {
+    it('should open create flyout when primary button is clicked', async () => {
       render();
       const addButton = await renderResult.findByTestId('testPage-emptyState-addButton');
 
       act(() => {
         userEvent.click(addButton);
       });
 
-      await waitFor(async () => {
-        expect(renderResult.getByTestId('testPage-flyout'));
-      });
+      expect(renderResult.getByTestId('testPage-flyout')).toBeTruthy();
+      expect(history.location.search).toMatch(/show=create/);
+    });
 
-      // indicate form is valid
-      act(() => {
-        const lastProps = getLastFormComponentProps();
-        lastProps.onChange({ item: { ...lastProps.item, name: 'some name' }, isValid: true });
+    describe('and the first item is created', () => {
+      it('should show the list after creating first item and remove empty state', async () => {
+        render();
+        const addButton = await renderResult.findByTestId('testPage-emptyState-addButton');
+
+        act(() => {
```

Comment:
no need to wrap with `act` here. You can also remove any other such `act` wrappers around `userEvent`

Reply:
you are [right](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily)! thanks!

c3a8d25